client/scripts/helpers/forms.coffee

```diff
@@ -83,40 +83,40 @@ class @Helpers.Client.Form
 
 
     @LimitFromToFields: (yearFrom, yearTo, monthFrom, monthTo) ->
-        limitEachOther = (from, to, extraCondition) ->
-            toValue = parseInt(to.val())
-            fromValue = parseInt(from.val())
-
-            to.find('option').each ->
+        hideOptionsLowerThan = (number, select) ->
+            select.find('option').each ->
                 currentValue = parseInt(@value)
-
-                doCheck = true
-
-                if extraCondition
-                    doCheck = extraCondition()
-
-                if doCheck and fromValue and currentValue and currentValue < fromValue
+                if currentValue and currentValue < number
                     $(@).hide()
                 else
                     $(@).show()
 
-            # in case a to date < from date was previously selected
-            if fromValue and toValue and toValue < fromValue
```

Comment:
Btw.: The values saved correctly after you change them. But when you reload the page "limit()" was triggered so it changed because of the bug mentioned above and auto-saved.